tox.ini

```
@@ -1,71 +1,64 @@
 [tox]
-env_list =
-    py{37,38,39,310,311,312}-test{,-devdeps,-numpy116,-numpy120,-numpy123}{,-cov}
+envlist =
+    py{39,310,311,312}-test{,-numpy122,-numpy125,-devdeps}{,-cov}
```

You ended up ditching numpy125 from the jobs.

```suggestion
    py{39,310,311,312}-test{,-numpy122,-devdeps}{,-cov}
```